src/main/java/com/redhat/red/build/koji/model/xmlrpc/KojiListTaggedRpmsParams.java

```diff
@@ -0,0 +1,181 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc. (jcasey@redhat.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.red.build.koji.model.xmlrpc;
+
+import org.commonjava.rwx.anno.DataKey;
+import org.commonjava.rwx.anno.StructPart;
+
+@StructPart
+public class KojiListTaggedRpmsParams
+                extends KojiParams
+{
+    @DataKey( "event" )
+    private Integer eventId;
+
+    @DataKey( "inherit" )
+    private Boolean inherit;
+
```

Comment:
This applies to other tag APIs as well (see `ListTaggedRequest`). The Koji API seems to indicate:

```
    set latest=True to get only the latest build per package
    set latest=N to get only the N latest tagged RPMs
```

So, this type here and in `ListTaggedRequest` should probably be a combined `Boolean` or `Integer`. However, I am not sure what I am supposed to see when I set, say, `latest=1`.

Reply:
I am not sure either. Maybe we just use the boolean latest if that is enough for your case. If people/user do want to get a few 'latest' in future, we can extend it then. I'd suggest adding a TODO comment here.

Reply:
I guess it would also work automatically as a plain `Object` and verifying `instanceof Boolean || Integer` when `setLatest()` is called?

Reply:
Maybe. You may want to change to Object and see what is generated in the xxx_parser class. And test it.

Reply:
I have added a `KojiBooleanOrInteger` object type based on the `KojiIdOrName` type in 6138bab.